team.go

```diff
@@ -26,6 +26,13 @@ type ListTeamOptions struct {
 	Query string `url:"query,omitempty"`
 }
 
+// AddUserToTeamOptions are the input parameters used when calling the AddUserToTeam API endpoint.
+type AddUserToTeamOptions struct {
+	TeamID string `json:"-"`
+	UserID string `json:"-"`
+	Role   string `json:"role,omitempty"`
```

Comment:
Based on your comment in the PR, there are only a handful of values that are accepted. Instead of this being a `string`, should we make it a named `string` type so that we can have a set of constants that people can also use?

```go
type UserRole string

const (
  UserRoleManager UserRole = "manager"
  // ...
)
```

The `User` struct also has a `Role` field that's a `string`. I'm not sure if it makes sense for these to be different. 🤔 

To be clear, this comment isn't a request to change but looking for your thoughts / feedback.